tidy3d/components/simulation.py

```diff
@@ -989,6 +991,25 @@ def _validate_monitor_size(self) -> None:
                 f"a maximum of {MAX_SIMULATION_DATA_SIZE_GB:.2f}GB are allowed."
             )
 
+    @cached_property
+    def monitors_data_size(self) -> typing.Dict:
```

Comment:
can we annotate this? so it would be `Dict[str, float]` I think?

Reply:
also, the convention we use throughout the rest of the code is
```py
from typing import Dict

...

... -> Dict[str, float]:
```
so this might be slightly better for consistency.

Reply:
done